app/controllers/api/authorizations_controller.rb

```diff
@@ -0,0 +1,13 @@
+# encoding: utf-8
+
+class Api::AuthorizationsController < Api::BaseController
+  include ApiAuthenticated
+  api_versions :v1
+  represent_with Api::AuthorizationRepresenter
+
+  private
+
+  def resource
+    Authorization.new(prx_auth_token)
```

Comment:
right! If we really need it, should look it up via API, but the token provides some of the same info, so I went with that.